yarn.lock

```diff
@@ -6450,6 +6455,11 @@ icss-utils@^2.1.0:
   dependencies:
     postcss "^6.0.1"
 
+idb@^4.0.5:
```

Comment:
@marcdumais is `ISC` fine? i hope it is using indexeddb api otherwise is very tedious :(

Reply:
yes, `ISC` is an Eclipse Foundation approved license